pontoon/base/models.py

```diff
@@ -1134,6 +1146,8 @@ class Project(AggregatedStats):
 
     tags_enabled = models.BooleanField(default=True)
 
+    is_dummy = models.BooleanField(default=False)
```

Comment:
Please add a comment explaining what a `dummy` project is. By the way, maybe we can find a better name than `dummy`? I'm not a big fan of it. Any suggestions?

Reply:
I am more and more confident that `dummy` is a bad name. We should find a better one. Here's what my brain can come up with: 

*Context: naming a type of projects that can be seen by everyone, exists for every language, but for which translations can only be seen by users who created them. We plan to use such a project to let users try features without having any impact on other users.*

`demo` -> if we're only ever using this for the one project that we'll use to show the features to users, then `demo` can work. 

`sandbox` -> conveys the fact that this project is a sandbox, meaning that what happens on that project has no impact to the rest of the world. It's more generic than `demo` and thus could be used for other potential use cases.

I'm actually quite satisfied with `sandbox`, but I'd like to get the opinion of @mathjazz and our Master of Naming Things, @stasm.

Reply:
To add a bit more context, we'll use this special project for the Pontoon Tour. The idea is that strings will guide the user through the features of Pontoon, like this: https://imgur.com/a/CQYvvad.

There is however another special project, which we'll use to demo in-context localization. Perhaps we can join them into one project?

These two projects should also not be visible in dashboards and excluded from stats.

I'm a bit skeptical about only exposing translation to the user that submitted them. I'm afraid it will make the code more complex on many different places, as is already seen in this PR. What if we instead don't store translations at all, and only keep them on the client side until next page load? In this case I'd vote for the name `demo`.

Reply:
I would be deeply confused if my translations were not kept between sessions, honestly. That's how smartcat does it, and it means you can come back to the project and use it as a reference.

We could figure out better ways to handle that special case, or refactor the code, to reduce the complexity, but I don't think we should make something weird because we fear the code would to complex.

Reply:
>I'm actually quite satisfied with sandbox, but I'd like to get the opinion of @mathjazz and our Master of Naming Things, @stasm.

_Is this a trap? Am I walking straight into it? I guess I am walking straight into it._

I see how _dummy_ is not a good word. I'm not sold on _demo_ either. To me a _demo_ is a time-limited or feature-limited version of the software. A demo is something to be shown during a presentation or in a screencast on the landing page. The way @mathjazz describes it, _tutorial_ might work well.

I also like _sandbox_, although it sounds rather specific about what happen to the translations. I.e. anyone can do anything and the result will not be saved, or will be saved but ignored ni production.

I realize that I'm probably not helping by adding yet another option, but if this is supposed to be a name of a _type_ of projects, then maybe _system projects_, _standard projects_, _builtin projects_, or _skeleton projects_ (a reference to `/etc/skel`) would work?

Thanks for looping me in and I hope I can help somehow. I'm happy to brainstorm other ideas but I'd like to make it clear that I'll be happy with any option you end up choosing.